lib/worker/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = { workspace = true }
 version = { workspace = true }
 
 [dependencies]
+actix-web = "4"
```

Comment:
do we want to add the dependency in the outer `cargo.toml` and use `{ workspace = true }` here. I don't know the correct way but this convention was followed earlier?

Reply:
👍 yeah, not really sure what's the convention. but we can keep following that